web/scripts/validator.dart

```diff
@@ -16,6 +16,8 @@
 
 // ignore_for_file: avoid_print
 
+// @dart=2.9
```

Comment:
Why does this say 2.9 when the range asks for 2.10?

Reply:
The docs suggest to use `2.9` as the language version here. Anyway, I'll remove it in the next release.